transportable-udfs-api/src/main/java/com/linkedin/transport/api/data/StdBytes.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2018 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.api.data;
+
+import java.nio.ByteBuffer;
+
+/** A Standard UDF data type for representing binary objects. */
+public interface StdBytes extends StdData {
+
+  /** Returns the underlying {@link ByteBuffer} value. */
+  ByteBuffer get();
+}
```

Comment:
Why did you choose ByteBuffer as the implementation?

Reply:
It's a wrapper of byte array. We want to work with the object instead of an array, so ByteBuffer is picked.

Avro also uses ByteBuffer for Binary type while Spark and Hive use byte array (easy to be wrapped to ByteBuffer)

Reply:
Seems like byte array is the best IR in this case (already native to Spark and Hive, and easily convertible to ByteBuffer, also more lightweight). Do you mind going with byte array? Also, as discussed offline, seems StdBinary makes more sense for the class name.

Reply:
Just noticed the class name was updated. Thanks for changing the StdBytes to StdBinary. May worth updating the PR title and description.

Reply:
As I said, ByteBuffer and byte array are the same thing. Please check out the implementation of ByteBuffer class.

As we do a lot of Java reflection code, a direct class like ByteBuffer is preferred to byte array.

Also replace "Bytes" by "Binary" in the last commit. Could you please check out the latest version?

Reply:
FYI, Pegasus and Avro use ByteBuffer.

Reply:
+1 for ByteBuffer. 
It allows for more efficient backing stores (e.g. off-heap impls are possible)

Reply:
Sounds good. ByteBuffer sounds like a reasonable choice.